Parser/pegen/pegen.c

```diff
@@ -703,7 +703,7 @@ Parser_New(struct tok_state *tok, int start_rule, int *errcode, PyArena *arena)
     return p;
 }
 
-mod_ty
```

Comment:
Why the change of type? Isn't `parse` returning always a `mod_ty`?

Reply:
> Why the change of type? Isn't parse returning always a mod_ty

No, in case `p->start_rule` is `Py_fstring_input`, `parse` calls `fstring_rule`, which directly returns an `expr_ty`.

> if not, we should update as well the type of `res`

Done!

Reply:
Every void* makes me sad... But I understand.